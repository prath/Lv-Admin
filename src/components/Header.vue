<template>
  <div>
    <!-- Header User Dashboard -->
    <header class="header header--user-dashboard">
      <div class="logo">
        <a href="#">
          <img
            src="../assets/img/logo-lokaventour.svg"
            alt
          >
        </a>
      </div>

      <div class="user-setting">
        <ul>
          <li>
            <a
              href="#"
              class="notif notif--top"
            >
              <img
                src="../assets/img/ic-notif.svg"
                alt
              >
            </a>
          </li>
          <li>
              <a v-on:click="logout">
                <div class="avatar avatar--extra-small">
                  <img
                    src="../assets/img/avatar.png"
                    alt
                  >
                </div>
              </a>
          </li>
        </ul>
      </div>
    </header>
    <!-- End Header User Dashboard -->

    <aside class="column side-icon">
      <nav>
        <ul>
          <li>
            <a href="#">
              <img
                src="../assets/img/ic-dashboard.svg"
                alt
              >
            </a>
          </li>
          <li>
            <a
              href="#"
              class="notif notif--center"
            >
              <img
                src="../assets/img/ic-wallet.svg"
                alt
              >
            </a>
          </li>
          <li>
            <a href="#">
              <img
                src="../assets/img/ic-analytic.svg"
                alt
              >
            </a>
          </li>
          <li>
            <a
              href="#"
              class="notif notif--center"
            >
              <img
                src="../assets/img/ic-comment.svg"
                alt
              >
            </a>
          </li>
        </ul>

        <ul class="bottom">
          <li>
            <a href="#">
              <img
                src="../assets/img/ic-help.svg"
                alt
              >
            </a>
          </li>
          <li>
            <a href="#">
              <img
                src="../assets/img/ic-settings.svg"
                alt
              >
            </a>
          </li>
        </ul>
      </nav>
    </aside>
    <aside class="column side-menu">
      <nav>
        <ul>
          <li>
            <router-link to="/dashboard">
              Dashboard
            </router-link>
          </li>
          <li>
            <router-link to="/users">
              Users
            </router-link>
          </li>
          <li>
            <router-link to="/tour-packages">
              Tour Packages
            </router-link>
          </li>
          <li>
            <router-link to="/featured-contents">
              Featured Contents
            </router-link>
          </li>
          <li>
            <router-link to="/tour-preferences">
              Tour Preferences
            </router-link>
          </li>
          <li>
            <router-link to="/categories-tags">
              Categories & Tags
            </router-link>
          </li>
          <li>
            <router-link to="/booking-contact">
             Booking by Contact List
            </router-link>
          </li>
          <li>
            <router-link to="/booking-list">
             Booking Orders
            </router-link>
          </li>
          <li>
            <router-link to="/refund-list">
              Refund Management
            </router-link>
          </li>
          <li>
            <router-link to="/financial-report">
              Financial Report
            </router-link>
          </li>
        </ul>
      </nav>
    </aside>
  </div>
</template>

<script>
  export default {
    methods: {
     logout: function () {

        localStorage.removeItem('accessToken');
        localStorage.removeItem('hostId');
        this.$router.push({ path: '/' })
      }
    }
  }
  </script>