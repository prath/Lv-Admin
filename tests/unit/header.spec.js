import { mount } from '@vue/test-utils'
import Header from '@/components/Header.vue'

describe('Header Component show ', () => {
  test('Header renders correctly', () => {
    const wrapper = mount(Header)
    expect(wrapper.element).toMatchSnapshot()
  })
})
