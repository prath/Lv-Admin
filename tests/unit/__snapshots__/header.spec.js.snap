// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component show  Header renders correctly 1`] = `
<div>
  <header
    class="header header--user-dashboard"
  >
    <div
      class="logo"
    >
      <a
        href="#"
      >
        <img
          alt=""
          src="assets/img/logo-lokaventour.png"
        />
      </a>
    </div>
     
    <div
      class="user-setting"
    >
      <ul>
        <li>
          <a
            class="notif notif--top"
            href="#"
          >
            <img
              alt=""
              src="assets/img/ic-notif.svg"
            />
          </a>
        </li>
         
        <li>
          <a
            href="#"
          >
            <div
              class="avatar avatar--extra-small"
            >
              <img
                alt=""
                src="assets/img/avatar.png"
              />
            </div>
          </a>
        </li>
      </ul>
    </div>
  </header>
   
  <aside
    class="column side-icon"
  >
    <nav>
      <ul>
        <li>
          <a
            href="#"
          >
            <img
              alt=""
              src="assets/img/ic-dashboard.svg"
            />
          </a>
        </li>
         
        <li>
          <a
            class="notif notif--center"
            href="#"
          >
            <img
              alt=""
              src="assets/img/ic-wallet.svg"
            />
          </a>
        </li>
         
        <li>
          <a
            href="#"
          >
            <img
              alt=""
              src="assets/img/ic-analytic.svg"
            />
          </a>
        </li>
         
        <li>
          <a
            class="notif notif--center"
            href="#"
          >
            <img
              alt=""
              src="assets/img/ic-comment.svg"
            />
          </a>
        </li>
      </ul>
       
      <ul
        class="bottom"
      >
        <li>
          <a
            href="#"
          >
            <img
              alt=""
              src="assets/img/ic-help.svg"
            />
          </a>
        </li>
         
        <li>
          <a
            href="#"
          >
            <img
              alt=""
              src="assets/img/ic-settings.svg"
            />
          </a>
        </li>
      </ul>
    </nav>
  </aside>
   
  <aside
    class="column side-menu"
  >
    <nav>
      <ul>
        <li
          class="active"
        >
          <router-link
            to="/"
          >
            
            Dashboard
          
          </router-link>
        </li>
         
        <li>
          <router-link
            to="/users"
          >
            
            Users
          
          </router-link>
        </li>
         
        <li>
          <router-link
            to="/tour-packages"
          >
            
            Tour Packages
          
          </router-link>
        </li>
         
        <li>
          <router-link
            to="/curated-contents"
          >
            
            Curated Contents
          
          </router-link>
        </li>
         
        <li>
          <router-link
            to="/tour-preferences"
          >
            
            Tour Preferences
          
          </router-link>
        </li>
         
        <li>
          <router-link
            to="/categories-tags"
          >
            
            Categories & Tags
          
          </router-link>
        </li>
         
        <li>
          <router-link
            to="/refund-management"
          >
            
            Refund Management
          
          </router-link>
        </li>
      </ul>
    </nav>
  </aside>
</div>
`;
